app/src/main/java/org/cru/godtools/ui/tools/ToolsAdapter.kt

```diff
@@ -51,13 +59,15 @@ class ToolsAdapter(lifecycleOwner: LifecycleOwner, private val dataModel: ToolsA
         tools = t
     }
 
-    override fun getItemViewType(position: Int) = when (getItem(position)?.type) {
-        Tool.Type.LESSON -> VIEW_TYPE_LESSON
+    override fun getItemViewType(position: Int) = when {
+        getItem(position)?.type == Tool.Type.LESSON -> VIEW_TYPE_LESSON
+        itemLayout == R.layout.dashboard_list_item_tools -> VIEW_TYPE_ALL_TOOL
```

Comment:
reverse the order of these and make the first condition `itemLayout != null -> VIEW_TYPE_CUSTOM`